expan/core/early_stopping.py

```diff
@@ -260,8 +260,10 @@ def bayes_factor(x, y, distribution='normal', num_iters=25000):
         dictionary with statistics
     """
     traces, n_x, n_y, mu_x, mu_y = _bayes_sampling(x, y, distribution=distribution, num_iters=num_iters)
-    kde = gaussian_kde(traces['delta'])
+    trace_to_evaluate_bf = traces['alpha'] if distribution == 'normal' else traces['delta']
+    trace_to_compute_credible_interval = traces['delta']
```

Comment:
The problem is in the models, ```normal.stan``` has ```alpha``` and ```delta```, but cauchy.stan only has ```delta```. I renamed the paras such that ```alpha``` is always the normalized ```delta```.

As mentioned by Jie, we want to select the bayes factor threshold based on the normalized para (```alpha```), while in the result we give credible intervals of the absolute effect size  (```delta```).